src/hooks/viewStateFacet/useGetArchaeologistAccusals.ts

```diff
@@ -0,0 +1,13 @@
+import { useContractRead } from 'wagmi';
+import { ViewStateFacetABI } from '../../abis/ViewStateFacet';
+
+export function useGetArchaeologistAccusals({ archaeologist }: { archaeologist: string }) {
+  const { data } = useContractRead({
```

Comment:
Is it worth it to make a useContractReadViewStateFacet(), that would have the address and abi, then would add functions and args here?

Reply:
Yes I think that would be a good idea